website/content/platform/development/contributing/how-to/add_data_to_existing_endpoint.md

```diff
@@ -0,0 +1,860 @@
+---
+title: How To Add A Data Provider To An Existing Endpoint
+sidebar_position: 2
+description: This guide outlines the process for adding an endpoint to an existing data provider and router endpoint.
+keywords:
+- OpenBB Platform
+- Open source
+- Python interface
+- REST API
+- contribution
+- contributing
+- documentation
+- code
+- provider
+- data
+- endpoint
+- existing
+- OpenBB extensions
+- OpenBB provider
+- standard model
+- provider model
+- how to
+---
+
+import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
+
+<HeadTitle title="How To Add A Data Provider To An Existing Endpoint - Contributing - Development | OpenBB Platform Docs" />
+
+This page will walk through adding a data provider to an existing endpoint, using a standard model. At a high level, the process will look something like:
+
+- Catalogue the parameters and returned fields from the chosen data provider.
```

Comment:
Is there a better term than "Catalogue"? (identify?)

Reply:
"document"?